Symfony/CS/Tests/Tokenizer/TokensAnalyzerTest.php

```diff
@@ -38,8 +38,14 @@ public function bar4()
 
         return " ({$a})";
     }
+
     public function bar5($data)
     {
+        $message = $data;
+        $example = function ($arg) use ($message) {
+            echo $arg . ' ' . $message;
+        };
+        $example('hello');
```

Comment:
Ow, thank you @GrahamCampbell ;)
@SpacePossum, could you make that rebase? It would be great to see only changes that will indeed be merged here ;)